src/Client/Updater.php

```diff
@@ -445,7 +445,7 @@ private function searchDelegatedRolesForTarget(TargetsMetadata $targetsMetadata,
                 // in the delegations from $targetsMetadata.
                 if ($delegatedRole->isTerminating()) {
                     $terminated = true;
-                    // § 5.6.7.2.2
+                    // § 5.6.7.2.1
```

Comment:
There still some lines in this branch that refer to `5.6.7.2.2` that have not been changed

Reply:
Thanks for pointing that out.

It looks like most of the references are actually still valid. But I did remove a reference to 5.6.7.2.3, which no longer exists.

Reply:
I see lines like "`// See § 5.6.7.2.1 and 5.6.7.2.2.`"

since 2.1 and 2.2 are not different. Then this now refers to different points in .19. Were they wrong before?

Reply:
My interpretation is that they weren't wrong; they were merely very general. I think it was our way of saying "we are dealing with delegation stuff".

I suspect they were wrong before, in a subtle way. The only point in Updater that mentioned 5.6.7.2.1 had this comment below it: "Recursively search the list of delegations in order of appearance." There was, and is, no apparent support for multi-role delegations (whatever those are).